csbmysql/csbmysql_suite_test.go

```diff
@@ -0,0 +1,100 @@
+package csbmysql_test
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+const (
+	bindingUserName = "binding-user"
+	bindingUserPass = "binding-'pass''word'"
+	adminUser       = "root"
+	adminPass       = "change-me"
+	dbHost          = "127.0.0.1"
+	bindingHost     = "%"
+	port            = 3306
+	database        = "nuclear-flux"
+)
+
+var (
+	adminUserURI = fmt.Sprintf("%s:%s@tcp(%s:%d)/mysql", adminUser, adminPass, dbHost, port)
+)
+
+func TestTerraformProviderCSBMySQL(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Terraform Provider CSB MySQL Suite")
+}
+
+var _ = BeforeSuite(func() {
+	By("Building provider")
+	mustRun("go", "build", "..")
+
+	By("Starting MySQL server")
+	mysqlVersion, ok := os.LookupEnv("TEST_MYSQL_VERSION_IMAGE_TAG")
```

Comment:
If the version of MySQL matters, then GitHub Actions could be used to create a test matrix of different versions.

Reply:
Done